velox/vector/tests/VectorTest.cpp

```diff
@@ -3899,5 +3899,21 @@ TEST_F(VectorTest, testOverSizedArray) {
   EXPECT_THROW(BaseVector::flattenVector(constArray), VeloxUserError);
 }
 
+TEST_F(VectorTest, testSliceBuffer) {
```

Comment:
- remove 'test' prefix
- split this into 2 tests: sliceBigintBuffer, sliceBooleanBuffer

Reply:
Updated, thanks.